features.md

```diff
@@ -8,11 +8,35 @@
 
 * ### Registering Patients
 
-   1. Entering Patient Details
```

Comment:
Why did you remove the above header?

Reply:
Actually I added header tags to make the section clearer. [Here](https://github.com/openmrs/openmrs-android-client-user-guide/pull/16/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R13)